doc/conf.py

```diff
@@ -53,6 +53,10 @@
 copyright = u'2015-2017 Zephyr Project members and individual contributors.'
 author = u'many'
 
+if "ZEPHYR_BASE" not in os.environ:
+    sys.stderr.write("$ZEPHYR_BASE environment variable undefined.\n")
```

Comment:
maybe add, "Did you source zephyr-env.sh?" to that error message.

Reply:
this is all for the 1.10 branch, I just cherry picked the change from master and do not want to make any changes to the commit, if you want, you can submit the change against master